cypress/e2e/swap/swapFlowLogging.test.ts

```diff
@@ -13,6 +13,23 @@ describe('time-to-swap logging', () => {
     cy.get('#swap-currency-output .token-amount-input').type('1').should('have.value', '1')
     cy.get('#swap-currency-input .token-amount-input').should('not.have.value', '')
 
+    // Verify first swap action
+    cy.waitForAmplitudeEvent(SwapEventName.SWAP_FIRST_ACTION).then((event: any) => {
+      cy.wrap(event.event_properties).should('have.property', 'time_to_first_swap_action')
+      cy.wrap(event.event_properties.time_to_first_swap_action).should('be.a', 'number')
+      cy.wrap(event.event_properties.time_to_first_swap_action).should('be.gte', 0)
+    })
+
+    // Verify Swap Quote
+    cy.waitForAmplitudeEvent(SwapEventName.SWAP_QUOTE_FETCH).then((event: any) => {
+      cy.wrap(event.event_properties).should('have.property', 'time_to_first_quote_request')
+      cy.wrap(event.event_properties.time_to_first_quote_request).should('be.a', 'number')
+      cy.wrap(event.event_properties.time_to_first_quote_request).should('be.gte', 0)
+      cy.wrap(event.event_properties).should('have.property', 'time_to_first_quote_request_since_first_input')
```

Comment:
nit: can we also add to the second swap in the session test to make sure this event property is undefined

Reply:
done